### PR TITLE
Add unit tests for transaction mediator

### PR DESCRIPTION
**Pull request by prabushi**

### CODE REVIEW

File: modules/core/src/test/java/org/apache/synapse/mediators/transaction/TransactionMediatorTestCase.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
+package org.apache.synapse.mediators.transaction;
```
Comment (manoramahp) [inline modules/core/src/test/java/org/apache/synapse/mediators/transaction/TransactionMediatorTestCase.java]: Add license header

File: modules/core/src/test/java/org/apache/synapse/mediators/transaction/TransactionMediatorTestCase.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    }
+
+    /**
+     * Test for transaction mediator creation using TransactionFactory mediator
+     *
+     * @throws Exception
+     */
+    public void testTransactionMediatorCreation() throws Exception {
+        Assert.assertEquals(NEW_ACTION, newActionMediator.getAction());
+        Assert.assertEquals(COMMIT_ACTION, commitActionMediator.getAction());
```
Comment (manoramahp) [inline modules/core/src/test/java/org/apache/synapse/mediators/transaction/TransactionMediatorTestCase.java]: Add assert message

File: modules/core/src/test/java/org/apache/synapse/mediators/transaction/TransactionMediatorTestCase.java [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    }
+
+    /**
+     * Test for set action method
+     *
+     * @throws Exception
+     */
+    public void testSetActionMethod() throws Exception {
+        Assert.assertEquals(ROLLBACK_ACTION, rollbackActionMediator.getAction());
+        Assert.assertEquals(FAULT_NOTX_ACTION, faultActionMediator.getAction());
```
Comment (manoramahp) [inline modules/core/src/test/java/org/apache/synapse/mediators/transaction/TransactionMediatorTestCase.java]: Add assert message

File: modules/core/src/test/java/org/apache/synapse/mediators/transaction/TransactionMediatorTestCase.java (hunk 1 above)
Comment (prabushi) [reply modules/core/src/test/java/org/apache/synapse/mediators/transaction/TransactionMediatorTestCase.java]: Updated

File: modules/core/src/test/java/org/apache/synapse/mediators/transaction/TransactionMediatorTestCase.java (hunk 2 above)
Comment (prabushi) [reply modules/core/src/test/java/org/apache/synapse/mediators/transaction/TransactionMediatorTestCase.java]: Updated

File: modules/core/src/test/java/org/apache/synapse/mediators/transaction/TransactionMediatorTestCase.java (hunk 3 above)
Comment (prabushi) [reply modules/core/src/test/java/org/apache/synapse/mediators/transaction/TransactionMediatorTestCase.java]: Updated

**Pull request merged**